dexbot/strategies/base.py

```diff
@@ -593,6 +601,18 @@ def get_lowest_own_sell_order(self, orders=None):
         except IndexError:
             return None
 
+    def get_external_market_center_price(self):
+#      def get_external_market_center_price(self, exchange='gecko', market='BTC/USD',  suppress_errors=False):  
+            """ Returns the center price of market including own orders. 
+            :param bool | suppress_errors: 
+            :return: Market center price as float
+            """
+            print("getting market center price for BTC/USD at 6363.00")
+            ticker_price = 6363.00 # dummy price 
+            return ticker_price
+
+
+
```

Comment:
Too many blank lines, these are in quite a few places. Inside a class only one is allowed, outside class max 2.